### ISSUE TITLE
[Buttons] MDCFloatingButton+Theming.h links are difficult to use for screen reader users.

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#5914]
The MDCFloatingButton+Theming.h inline documentation has links to the Floating Button article on Material.io.  However, since the only text wrapped in the anchor is "Material Guidelines", screenreader users who are jumping between links on the rendered documentation will have difficulty realizing that it does not link to material.io directly, but to the component article for the Floating Action Button.

Caused by #5869

---
#### Internal data
- Associated internal bug: [b/120671450](http://b/120671450)

**Comment by andrewoverton** [comment]
@romoore, I'm a little confused about how it should be. Right now it's like this:
```
The [Material Guidelines article for Floating Action
 Buttons](https://material.io/design/components/buttons-floating-action-button.html#theming) has
 more details about how Floating Action Buttons can be styled and used.
```
What should the link text be and what should the link URL be?

**Comment by romoore** [comment]
Fixed in https://github.com/material-components/material-components-ios/pull/5927
